### PR TITLE
Add load resource plug to the readme

### PR DESCRIPTION
**Pull request by hassox**
Addresses https://github.com/ueberauth/guardian/issues/124

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -96,6 +96,18 @@ will call the `:unauthenticated` function of your handler.
 When you ensure a session, you must declare an error handler. This can be done
 as part of a pipeline or inside a Phoenix controller.
 
+### Guardian.Plug.LoadResource
+
+Up to now the other plugs have been just lookinf for valid tokens in various
```
Comment (doomspork) [inline README.md]: looking

Comment (hassox) [inline README.md]: eep... Thanks

**Comment by doomspork** [comment]
Looks good @hassox :+1:

File: README.md [hunk 2]
```diff
@@ -96,6 +96,18 @@ will call the `:unauthenticated` function of your handler.
 When you ensure a session, you must declare an error handler. This can be done
 as part of a pipeline or inside a Phoenix controller.
 
+### Guardian.Plug.LoadResource
+
+Up to now the other plugs have been just looking for valid tokens in various
+places or making sure that the token has the correct permissions.
+
+The `LoadResource` plug looks in the `sub` field of the token, fetches the
+resource from the Serializer and makes the available via
```
Comment (doomspork) [inline README.md]: "and makes it available"

File: README.md [hunk 3]
```diff
@@ -96,6 +96,18 @@ will call the `:unauthenticated` function of your handler.
...
+`Guardian.Plug.current_resource(conn)`
+
+Note that this does not _ensure_ that there is a resource that could be loaded.
```
Comment (doomspork) [inline README.md]: Maybe we can rewrite this to be a little clearer, something like: 

"Note:  This does _not ensure_ a resource will be loaded.  If there is no available resource, `current_resource/1` will return `nil`."

**Comment by doomspork** [comment]
@hassox I went back over this and found a few pieces of feedback, then we can get this one merged 👍

**Pull request merged**